src/main/resources/application.conf

```
@@ -33,6 +33,7 @@ backend-for-frontend {
     tenant-management = ${TENANT_MANAGEMENT_URL}
     tenant-process = ${TENANT_PROCESS_URL}
     purpose-process = ${PURPOSE_PROCESS_URL}
+    client-process = ${CLIENT_PROCESS_URL}
```

this was fine because it refers to the actual authorization process service
```suggestion
    authorization-process = ${AUTHORIZATION_PROCESS_URL}
```